### PR TITLE
Fix jazzy references on jazzy requierements.

### PR DESCRIPTION
**Pull request by marcoag**

### CODE REVIEW

File: config/jazzy/requirements/features-executable.yaml [hunk 1]
```diff
@@ -362,11 +362,11 @@ requirements:
     checks:
       - name: >-
           Check `gazebo --verbose
-          /opt/ros/rolling/share/gazebo_plugins/worlds/gazebo_ros_camera_distortion_barrel_demo.world`
+          /opt/ros/jazzy/share/gazebo_plugins/worlds/gazebo_ros_camera_distortion_barrel_demo.world`
```
Comment (Yadunund) [inline config/jazzy/requirements/features-executable.yaml:365]: You might also want to go through all the requirements and update the gazebo classic references to gazebo (gz).

**Review by Yadunund — APPROVED** [review]
LGTM. Left a comment to consider for this PR or in the next one.

File: config/jazzy/requirements/features-executable.yaml (hunk 1 above)
Comment (clalancette) [reply config/jazzy/requirements/features-executable.yaml:365]: Yeah, that's an excellent point.  We'll definitely need to do that.  Maybe @azeey can help here?

**Review by clalancette — APPROVED** [review]
While I do think we should update the Gazebo tests as suggested by Yadu, I'm approving anyway since this is strictly an improvement.

File: config/jazzy/requirements/features-executable.yaml (hunk 1 above)
Comment (marcoag) [reply config/jazzy/requirements/features-executable.yaml:365]: Great point, thanks! will merge this first and we can do that on a separate PR.

Comment (azeey) [reply config/jazzy/requirements/features-executable.yaml:365]: I started looking at this but I was confused about the command `ros2 run gazebo_ros_pkgs gazebo`. Did that ever work? I don't see any executables in the [package](https://github.com/ros-simulation/gazebo_ros_pkgs/tree/ros2/gazebo_ros_pkgs). 

Also, is this meant to just check that Gazebo is working? i.e, can I just use a simple world that gets shipped with Gazebo?

**Pull request merged**